core/ts/MaciState.ts

```diff
@@ -38,7 +38,6 @@ interface BatchSizes {
 }
 
 interface MaxValues {
-    maxUsers: number;
```

Comment:
Why remove `maxUsers`?

Reply:
another type error + this is just not a valid value to insert there. The max number of users is determined by the depth of the state tree; also this maci-core is supposed to mirror the smart contracts, and in the smart contracts the MaxValues struct does not have maxUsers

Reply:
Gotcha, thanks!